infrastructure/local-development/local.linux.yml

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's call this file a `local.linux.yml.template`. Please update the @todo commends accordingly. Not sure if we even need them if user never has to touch the file manually

Reply:
Also make sure the provision script doesn't change the template but only copies it as a new file `local.linux.yml`